src/DropdownToggle.js

```diff
@@ -67,7 +67,7 @@ class DropdownToggle extends React.Component {
     ), cssModule);
     const children = props.children || <span className="sr-only">{ariaLabel}</span>;
 
-    if (nav) {
+    if (nav && Tag === Button) {
```

Comment:
Checking if the tag is the default... but what if set by the user and was another reactstrap component which could accept `tag`... should this code block be ran?
I was thinking about checking if `Tag` was a function and whether or not `Tag.propTypes.tag` was defined or something to see if it could accept this prop.
Let me know if that is something that is desired or if just checking that `Tag` is the default is good enough.